server/src/main/java/com/jungleapp/cs414/server/Match.java

```diff
@@ -1,4 +1,51 @@
 package com.jungleapp.cs414.server;
 
+import spark.Request;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
 public class Match {
-}
+
+    private JungleBoard board;
```

Comment:
This looks okay, but we are currently working on a version to simplify our code in general, which may change this a lot in the future, just a heads up that we may want to implement this class to better suit back end logic.